distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -135,6 +135,7 @@ ALERT;AndroidTV Binding: The thing configuration of 'port' has been renamed and
 ALERT;evcc Binding: Update to evcc API version 0.123.1 results in several new or updated channels. Existing Items will need to be adjusted.
 ALERT;ISM8 Binding: Most channels have changed and are now using Units of Measurements. Items must be adapted and the things must be recreated.
 ALERT;Jython Scripting: Default python lib path changed from "/automation/lib/python" to "/automation/jython/lib" and default python script path changed from "automation/jsr223" to "automation/jython". Just move your python scripts and libraries to new locations. The path "automation/jsr223" is still working, because it is used as a deprecated path for all automation add-ons.
+ALERT;Persistence Actions: historicState and evolutionRate have been deprecated and replaced by persistedState and evolutionRateSince. They may get removed in a future version. Return values of actions may now be QuantityType.
```

Comment:
```suggestion
ALERT;Persistence Extensions: "historicState" and evolutionRate" have been deprecated and replaced by "persistedState" and "evolutionRateSince". They will be removed in a future version. Methods may now return QuantityType instead of DecimalType. See documentation for details.
```

Reply:
adjusted